app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        issues_found = 0
+        if "vaccine" not in visitor:
+            issues_found += 1
+            raise NotVaccinatedError
+        else:
```

Comment:
The use of `else` after a `raise` statement is unnecessary. According to the checklist, avoid using unnecessary `else` blocks. You can directly check the condition without `else`.